lms/djangoapps/mobile_api/social_facebook/courses/tests.py

```diff
@@ -11,6 +14,7 @@
 from ..test_utils import SocialFacebookTestCase
 
 
+@patch.dict(settings.FEATURES, {"CUSTOM_COURSES_EDX": False})
```

Comment:
It should be Ok if it tries to get the custom course.  The call to `get_current_ccx` should return `None`.  We should have our unit tests run with the feature flags that are enabled by default to mimic production as close to possible.

Are you getting an exception?